### DISCUSSION TITLE
Support environment variables for key config items

### DISCUSSION THREAD

**Discussion by danpowell88** [ptr727/PlexCleaner#211] [Ideas]
It'd be great to have support for loading some of the key config items from env vars to make running under docker easier

The things I'm talking about would be the mode (process/monitor etc), log file, media files and settings file location, possibly also parallel.

Looks like it is possible by setting the defualt value to be loaded by environment vars (although not ideal in the way it integrates with the cmd line api)

https://github.com/dotnet/command-line-api/issues/1191

**Comment by ptr727** [comment]
Could you describe how using docker environment variables are easier, I'm having difficultly seeing how setting environment variables per container instance run is easier than setting commandline arguments per run?
